src/core/nativecall.c

```diff
@@ -372,8 +372,162 @@ static const char *dlerror(void)
 }
 #endif
 
+void init_c_call_node(MVMThreadContext *tc, MVMSpeshGraph *sg, MVMJitNode *node, void *func_ptr, MVMint16 num_args, MVMJitCallArg *args) {
```

Comment:
Does this bunch of code need to be conditionally compiled depending on whether or not we build the JIT?

Reply:
I can't find any conditional compilation of JIT code? Only the emit part but not the graph itself.

Reply:
What we do is 'stub' the JIT compiled code. The JIT graph declarations are always compiled, though, so this should be safe. We do not, in fact, have declared a `MVM_JIT_IS_COMPILED` parameter or something of that sort, so at worst, this is wasteful.